applications/StructuralMechanicsApplication/custom_elements/updated_lagrangian.h

```diff
@@ -152,8 +230,10 @@ class UpdatedLagrangian
     ///@name Protected member Variables
     ///@{
 
-    std::vector<double> mDetF0; // Historical total elastic deformation measure determinant
-    std::vector<Matrix> mF0; // Historical total elastic deformation measure
+    /* Historical total elastic deformation measure */
+    bool mF0Computed;           // To avoid computing more than once the historical total elastic deformation measure 
+    std::vector<double> mDetF0; // The historical total elastic deformation measure determinant
```

Comment:
F0 is the deformation gradient of the current configuration (or the total deformation gradient of the previous step)